README.md

```diff
@@ -1,148 +1,431 @@
-supported systems, or build your own binary from source if needed.
+You can just copy your world into the worlds directory:
 
-Repositories for automatic installation are also available:
-* [Debian/Ubuntu](http://overviewer.org/debian/info)
-* [RHEL/CentOS/Fedora](http://overviewer.org/rpms/info)
+    mkdir /opt/mscs/worlds/alpha  // Makes a directory called `alpha` in the `/worlds/` folder
+    cp -R * /opt/mscs/worlds/alpha // Copies the `alpha` world data folder into `/opt/mscs/worlds/alpha`
+    
+In sum, the ending path of your world `alpha` should be `/opt/mscs/worlds/alpha/alpha`.
 
-### Firewall / NAT
-If you have a firewall installed on your computer, or a router using NAT
-installed in your network, you will need to route some ports to your server.
-Instructions on how to accomplish this are beyond the scope of this post, but
-here are some things you will need to know:
-* The default port for the Minecraft server is: `25565`.
-* If you wish to run multiple world servers using this script, you will
-  want to open a range of ports (for example `25565 - 25575`).
+===================
+**Optional**: If you would like to rename the `alpha` folder 
+(the one that is the parent folder of the actual world) to a different name,
+follow the steps below. In this example we wanted to rename the `alpha` folder to `vanillaMC`:
 
-See the [iptables.rules](iptables.rules)
-file for a very basic set of rules that you can use with the Iptables firewall.
+    mkdir /opt/mscs/worlds/vanillaMC
+    cp -R * /opt/mscs/worlds/vanillaMC
+    mv /opt/mscs/worlds/vanillaMC/alpha /opt/mscs/worlds/vanillaMC/vanillaMC
+    mscs create vanillaMC 25565
+=================
 
+Make sure you check `server-port` and `query.port` in `server.properties` to make sure it does not overlap with other servers created by the MSCS script. Also ensure that `enable-query` is set to `true`.  If you do not have `enable-query` and a `query.port` set, you will not be able to check the status of the world with the script.
 
-## Usage
+After you've set up the file structure, you now need to create a world entry 
+into MSCS. Do this via:
+
+        mscs create [world] [port] <ip>
+  
+Where `world` is the world name
+(so it would be "world" from the previous example", 
+and `port` is the server port (by default, use 25565).
+`ip` is optional and will be used if you wish to create multiple worlds 
+across different servers. For now, leave it blank.
+
+Afterwards, start the server via `mscs start [world]`. 
+Then, accept the EULA, run the start command again, and you're good to go!
+
+As a last note, make sure you check `server-port` and `query.port` in 
+`server.properties` 
+to make sure it does not overlap with other servers created by the MSCS script. 
+Also ensure that `enable-query` is set to `true`.  
+If you do not have `enable-query` and a `query.port` set, 
+you will not be able to check the status of the world with the script.
```

Comment:
A lot of repetition here.  If the create, copy and rename all mention running `mscs create`, there is no need to include another section.  The EULA section should probably be right there with create, since copy and rename are probably functional servers already.  Since we are telling users that they should run `mscs create`, then we probably don't have to also mention verifying the `server-port`, `query.port`, `etc` since that is the job of `mscs create`.